django/contrib/auth/hashers.py

```diff
@@ -63,11 +61,32 @@ def check_password(password, encoded, setter=None, preferred="default"):
     if not is_correct and not hasher_changed and must_update:
         hasher.harden_runtime(password, encoded)
 
+    return is_correct, must_update
+
+
+def check_password(password, encoded, setter=None, preferred="default"):
+    """
+    Return a boolean of whether the raw password matches the three
+    part encoded digest.
+
+    If setter is specified, it'll be called when you need to
+    regenerate the password.
+    """
+    is_correct, must_update = verify_password(password, encoded, preferred=preferred)
+
     if setter and is_correct and must_update:
         setter(password)
     return is_correct
 
 
+async def acheck_password(password, encoded, asetter=None, preferred="default"):
```

Comment:
Both functions should have the same signature
```suggestion
async def acheck_password(password, encoded, setter=None, preferred="default"):
```